ingestion-scheduler/src/main/java/com/datahub/metadata/ingestion/IngestionScheduler.java

```diff
@@ -204,6 +204,9 @@ public BatchRefreshSchedulesRunnable(
     public void run() {
       try {
 
+        // First un-schedule all currently scheduled runs (to make sure consistency is maintained)
+        _unscheduleAll.run();
```

Comment:
so this unschedules all runs?

Reply:
on startup?

Reply:
Well yeah but also on a periodic refresh we do